pkg/graph/schema.graphql

```diff
@@ -25,6 +26,15 @@ type System {
   name: String!
 }
 
+"""
+A document that belongs to an accessibility request
+"""
+type AccessibilityRequestDocument {
+  id: UUID!
+  name: String!
+  uploadedAt: String!
```

Comment:
I think we want to use a `Time` scalar here:

```suggestion
  uploadedAt: Time!
```

Reply:
I also wonder if we should use `createdAt` instead of `uploadedAt` for consistency? Will those values ever both exist and be different?

Reply:
Thanks for pointing me to the new scalar! At the database level, I think we should def use createdAt. I was thinking we could maybe expose that as uploadedAt, similar to how the 508 request exposes createdAt as submittedAt. But open to suggestions.